src/components/piece/piece.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState} from 'react';
+import './piece.css';
+
+
+
```

Comment:
Now that you've merged the lint settings, run the 'format' script to format your code.